File: src/main/java/me/miki/shindo/management/notification/NotificationType.java
package me.miki.shindo.management.notification;

import me.miki.shindo.management.nanovg.font.LegacyIcon;

public enum NotificationType {
    INFO(LegacyIcon.INFO),
    WARNING(LegacyIcon.ALERT_TRIANGLE),
    ERROR(LegacyIcon.X_CIRCLE),
    SUCCESS(LegacyIcon.CHECK),
    MUSIC(LegacyIcon.MUSIC);

    private final String icon;

    NotificationType(String icon) {
        this.icon = icon;
    }

    public String getIcon() {
        return icon;
    }
}

File: src/main/kotlin/me/miki/shindo/management/notification/NotificationType.kt
package me.miki.shindo.management.notification

import me.miki.shindo.management.nanovg.font.LegacyIcon

enum class NotificationType(val icon: String) {
    INFO(LegacyIcon.INFO),
    WARNING(LegacyIcon.ALERT_TRIANGLE),
    ERROR(LegacyIcon.X_CIRCLE),
    SUCCESS(LegacyIcon.CHECK),
    MUSIC(LegacyIcon.MUSIC)
}
